MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -49396,10 +49556,16 @@
       "RequiredItems": [
         "AreaSnowheadTempleAccess",
         "Enter Snowhead Temple",
-        "OtherArrow",
         "SHT Icicle Room Access"
       ],
-      "ConditionalItems": [],
+      "ConditionalItems": [
+        [
+          "OtherArrow"
+        ],
+        [
+          "Zora Boomerang Through Walls"
+        ]
```

Comment:
Do we want to also add getting these with something like Goron's giant hitbox while rolling? Or sometimes you can just climb up into the ice and get the rupee as human link.

Reply:
Would be a nice trick to add in the future, since it could also apply to the rupees in the pre-boss room of ISTT. Grabbing the items as just human can be added with the new trick Fewer Item Requirements though.